packages/core/src/util/esm.js

```diff
@@ -6,15 +6,6 @@
 
 const semver = require('semver');
 
-/**
- * Check if the given Node.js version supports ESM.
- * @param {string} version - The Node.js version to check.
- * @returns {boolean} - True if ESM is supported, false otherwise.
- */
-exports.esmSupportedVersion = function esmSupportedVersion(version) {
```

Comment:
qs: Do you remember why we have added this function in the first place and not used `packages/core/src/tracing/supportedVersion.js`?

Reply:
If I remember correctly, when we added this file, we were supporting Node.js v10 and v12, with ES module support added in v14 and above. There was also a known issue with v20. Therefore, we have a specific range of supported Node.js versions, which is why we kept in a separate file for reuse.

Reply:
Yes that makes sense. Thank u